staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -85,6 +85,14 @@ type objState struct {
 	stale bool
 }
 
+func (s *store) Count(pathPrefix string) (int64, error) {
```

Comment:
I'd expect the key parameter to match that passed to GetToList(), etc, and get joined with the pathPrefix already held by `store`

```
func (s *store) Count(key string) (int64, error) {
  key = path.Join(s.pathPrefix, key)
  getResp, err := s.client.KV.Get(context.Background(), key, ...
  ...
}
```

Reply:
Done.